Ghidra/Framework/Utility/src/main/java/utility/module/ModuleManifestFile.java

```diff
@@ -95,7 +95,16 @@ else if (trimmedLine.startsWith(DEPENDENCY_IDENTIFIER)) {
 			// ignore for now
 		}
 		else if (trimmedLine.startsWith(EXCLUDE_FROM_GHIDRA_JAR)) {
-			excludeFromGhidraJar = true;
+			String[] tokens = trimmedLine.split(":");
+
+            if (tokens.length == 2) {
+                if (tokens[1].toLowerCase().trim().compareTo("false") == 0)
```

Comment:
I think the default should be to not exclude, so only exclude if true was used.  If we do that, you can just do:

```
String[] tokens = trimmedLine.split(":");
String value = tokens.length == 2 ? tokens[1].trim() : "";
excludeFromGhidraJar = Boolean.valueOf(value);
```

Reply:
Sure, thanks